libs/host/Configuration/Options.cs

```diff
@@ -339,6 +339,10 @@ internal sealed class Options
         [Option("maxthreads", Required = false, HelpText = "Maximum worker and completion threads in thread pool, 0 uses the system default.")]
         public int ThreadPoolMaxThreads { get; set; }
 
+        [IntRangeValidation(0, int.MaxValue)]
+        [Option("connection-limit", Required = false, Default = 0, HelpText = "Maximum number of client connections")]
+        public int ConnectionLimit { get; set; }
```

Comment:
will this be inclusive of Gossip connections incase of cluster mode?

Reply:
yes, you have to account for the internode connections, which means gossip, replication, failover and migration connections. The failover and migration are transient connections until its operation finishes. Gossip and replication and long-lasting connections.